app/src/main/java/com/example/easycloset/SplashScreenActivity.java

```diff
@@ -2,13 +2,26 @@
 
 import androidx.appcompat.app.AppCompatActivity;
 
+import android.content.Intent;
 import android.os.Bundle;
+import android.os.Handler;
 
 public class SplashScreenActivity extends AppCompatActivity {
+    Handler handler;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_splashscreen);
+
+        handler = new Handler();
+        handler.postDelayed(new Runnable() {
+            @Override
+            public void run() {
+                Intent intent = new Intent(SplashScreenActivity.this, LoginActivity.class);
+                startActivity(intent);
+                finish();
+            }
+        }, 3000);
```

Comment:
I assume that if user will close SplashScreenActivity (e.g. by pressing back) before 3000 elapsed then LoginActivity will still open. You may pause or cancel your handler by overriding onPause/onStop/onDestroy

Reply:
working on this

Reply:
fixed

Reply:
how did you fix it?

Reply:
i used timer instead of handler becuase we can cancel the Timer instance in either onPause()or onDestroy() methods to stop the scheduled launch of the home / login screens.

Reply:
thanks, noted, left a comment for it